x-pack/plugins/maps/public/elasticsearch_geo_utils.js

```diff
@@ -101,31 +104,28 @@ export function geoPointToGeometry(value) {
     // Geo-point expressed as an array with the format: [lon, lat]
     const lat = value[1];
     const lon = value[0];
-    return [pointGeometryFactory(lat, lon)];
+    accumulator.push(pointGeometryFactory(lat, lon));
+    return accumulator;
   }
 
   // Geo-point expressed as an array of values
-  return value.reduce(
-    (points, itemInValueArray) => {
-      return points.concat(geoPointToGeometry(itemInValueArray));
-    },
-    []
-  );
+  for (let i = 0; i < value.length; i++) {
+    accumulator.push(geoPointToGeometry(value[i]));
+  }
+  return accumulator;
 }
 
-export function geoShapeToGeometry(value) {
+export function geoShapeToGeometry(value, accumulator = []) {
   if (!value) {
-    return [];
+    return accumulator;
```

Comment:
Returns are not longer being used. Not need to return anything from this function